### ISSUE TITLE
only one taxon_id and option_type_id possible when creating product

### ISSUE THREAD

**Issue by hefan** [solidusio/solidus#2791]
When you create a product and try to add more than one taxon_id or more than one option_type_id only the first one is saved.

During Update it works as it should.

### Steps to reproduce
Create a new product and add more than one option_type_id or taxon_id to it. Click Create.

### Expected behavior
The new Product should be created with more than one option type or taxon assigned

### Actual behavior
Only the first option type or taxon is assigned. you need to assign the rest by updating the product again.

### System configuration
**Solidus Version**:
v2.6.0

**Extensions in use**:
* solidus (2.6.0)
* solidus_api (2.6.0)
* solidus_auth_devise (2.1.0)
* solidus_backend (2.6.0)
* solidus_core (2.6.0)
* solidus_frontend (2.6.0)
* solidus_i18n (2.0.0)
* solidus_menus (1.0)
* solidus_sample (2.6.0)
* solidus_static_content (1.0.0 eae8e82)
* solidus_support (0.2.1)

### Possible Solution

Just Split the taxon_ids and option_type_ids params similar like in the update function:
https://github.com/solidusio/solidus/blob/master/backend/app/controllers/spree/admin/products_controller.rb

Something like:

```
before_action :split_params, only: [:create]

  def split_params
    if params[:product][:taxon_ids].present?
      params[:product][:taxon_ids] = params[:product][:taxon_ids].split(',')
    end
    if params[:product][:option_type_ids].present?
      params[:product][:option_type_ids] = params[:product][:option_type_ids].split(',')
    end
  end
```

**Comment by jacobherrington** [comment]
I am able to replicate this issue - planning on taking a look at it later today.

**Comment by jacobherrington** [comment]
Fix for this in #2840 pending review.

**Comment by jacobherrington** [comment]
@kennyadsl can this issue be closed now that you have merged #2840?

**Comment by kennyadsl** [comment]
Sure, thanks for pointing out!
